content/developer/reference/backend/views.rst

```diff
@@ -976,6 +976,37 @@ system. Available semantic components are:
   ``attrs``
     same as for ``field`` component.
 
+  .. _reference/views/form/setting:
+``setting``
+  The ``setting`` tag is used to declare a setting. The first field in the setting is
```

Comment:
The ``setting`` tag is used to declare a setting, an entity visually separated on two panels (left and right), and used to edit a given field.